src/hooks/useSignedIn.tsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from 'react';
+import { Auth } from 'aws-amplify';
+import { useHistory } from 'react-router-dom';
+
+export const useSignedIn = () => {
+	const [signedIn, setSignedIn] = useState(false);
+	const history = useHistory();
+	const signOut = () => {
+		Auth.signOut();
+		setSignedIn(false);
+		history.go(0);
+	};
+
+	const checkAuthenticated = () => {
+		Auth.currentAuthenticatedUser()
+			.then((user) => {
+				console.log(user);
```

Comment:
I assume this was used for debugging, we don't want it in master.

Reply:
We had console.logs as placeholders. When we are ready for production, I will enforce this. Ticket made!